webapp/channels/src/components/global_header/left_controls/product_menu/product_branding_team_edition/product_branding_team_edition.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import styled from 'styled-components';
+
+import Logo from 'components/common/svg_images_components/logo_dark_blue_svg';
+
+const ProductBrandingTeamEditionContainer = styled.div`
+    display: flex;
+    align-items: center;
+    
+    svg,
+    svg path {
```

Comment:
The idiomatic way to do this is to do an override fro the Logo component. So:
```
const StyledLogo = styled(Logo)`
... styling here
`
```

Reply:
I tried and this is not working with SVG, even if I allow to pass the `className` to the component and pass it to the root `svg` element, it still doesn't work. So the current solution works, and I think is localized enough to not be a problem.

Reply:
The pattern does work: https://github.com/mattermost/mattermost-plugin-ai/blob/2d13eb69da506ca83e1845bc6f309fc2b42b86f6/webapp/src/components/assets/icon_sparkle_question.tsx#L6